site/search/Autocomplete.tsx

```diff
@@ -198,14 +190,6 @@ const AlgoliaSource: AutocompleteSource<BaseItem> = {
                         distinct: true,
```

Comment:
Wondering if we should switch out `Charts` here for `ExplorerViewsMdimViewsAndCharts`, and maybe increase `hitsPerPage` to 3 also, to get a kind-of-similar result to before?